test/integration/targets/refresh_inventory/runme.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+set -eux
```

Comment:
The tests are not running yet :smile:

Reply:
they don't pass yet

Reply:
was surprised ci was green after playing around with it, so pointed it out in case you forgot to commit a change